src/main/java/org/skriptlang/skript/bukkit/tags/elements/ExprTagContents.java

```diff
@@ -0,0 +1,87 @@
+package org.skriptlang.skript.bukkit.tags.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.bukkitutil.EntityUtils;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Keywords;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.Material;
+import org.bukkit.Tag;
+import org.bukkit.entity.EntityType;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Objects;
+
+@Name("Tags Contents")
+@Description({
+		"Returns all the values that a tag contains.",
+		"For item and block tags, this will return items. For entity tags, " +
+		"it will return entity datas (a creeper, a zombie)."
+})
+@Examples({
+		"broadcast tag values of minecraft tag \"dirt\"",
+		"broadcast (first element of player's tool's block tags)'s tag contents"
+})
+@Since("INSERT VERSION")
+@Keywords({"blocks", "minecraft tag", "type", "category"})
+public class ExprTagContents extends SimpleExpression<Object> {
+
+	static {
+		Skript.registerExpression(ExprTagContents.class, Object.class, ExpressionType.PROPERTY,
+				"tag (contents|values) of %minecrafttag%",
+				"%minecrafttag%'[s] tag (contents|values)");
```

Comment:
I thought it would get a bit confusing, having multiple items' tags concatenated into one list. Do you think that's a concern?

Reply:
No because it's user defined. If there are multiple tags provided return multiple. Similarity to Property expressions automatically determining the isSingle

Reply:
Yeah, but with property expressions it's 1 to 1. Here, it's 1 to many, so it's impossible to tell which contents were from which tag. Plus whether duplicate items should be included, or if it should just be 1 instance of each material.
Let me know what you think.

Reply:
Ok I see, do you think the ExprTag should be called ExprTags and return multiples then since it's a collection of objects? I see contents being single.

Reply:
Yeah, exprtags could be made plural